### PR TITLE
Add tests for font loading behavior

### PR DESCRIPTION
**Pull request by i2amsam**
Add tests for existing font loading an the new fixed-size from #107

### CODE REVIEW

File: spec/fixtures/fonts/fake_cjk_font.otf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# This file here to allow tests to load example configurations.
+私はフォントではないです。
```
Comment (mfurtak) [inline spec/fixtures/fonts/fake_cjk_font.otf]: ああ！ほんとうだ。

File: spec/editor_spec.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        editor = Frameit::Editor.new
+
+        editor.screenshot = screenshot
+
+        # TODO: This should probably be a collaborator instead a mock
+        expect(editor).to receive(:fetch_config).at_least(:once).and_return config
+
+        editor
+      end
```
Comment (mfurtak) [inline spec/editor_spec.rb]: move to `spec_helper`?

**Comment by mfurtak** [comment]
:raised_hands: :raised_hands: :raised_hands: for tests!

File: spec/editor_spec.rb (hunk 2 above)
Comment (i2amsam) [inline spec/editor_spec.rb]: I'll move it if it ends up useful in more than just this file.

**Comment by KrauseFx** [comment]
This is awesome, I'm super excited about the recent `frameit` cleanups :+1:

**Pull request merged**